tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm

```diff
@@ -2,11 +2,20 @@
 //  RunModelViewController.m
```

Comment:
You should pull these first few lines. Please see https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#license for documentation on how the header/license should go. I would use https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/op.cc#L1 as an example.